Makefile

```diff
@@ -59,6 +59,7 @@ test: run-unit-tests
 
 .PHONY: lint
 lint: build-slim build-test
+	# Note: Make sure that you run `git config diff.noprefix false` in this repo
```

Comment:
Actually, we could link here to upstream issue, as this is not desired behavior.

Reply:
https://github.com/golangci/golangci-lint/issues/948

Reply:
done